### DISCUSSION TITLE
Tile Overlay using local tiles : not showing

### DISCUSSION THREAD

**Discussion by SebastienLinsFlyx** [react-native-maps/react-native-maps#4599] [Q&A]
Hello,

I am using react-native-maps to display a map with a custom Tile Overlay.
It is working when I use <UrlTile urlTemplate={'myUrl/Tiles/{z}/{x}/{y}.jpg'} /> and my custom tiles are shown.
Now I want to use the <LocalTile /> way.
I want to store the details in the application and display them but I don't understand where to place them and the path to them.
I have try to put the in a assets file a do the <<LocalTile /> like this 
  <LocalTile
            pathTemplate={'./assets/Tiles/{z}/{x}/{y}.jpg'}
          />
But it is not working.

How can I manage this ?

Thank you

![tiles](https://user-images.githubusercontent.com/124349558/216562810-921fa2a5-58af-461b-b47d-5a1174002f19.png)

**Comment by ilias-jabri** [comment]
have you found the solution? I'm facing the same issue right now.

**Reply by SebastienLinsFlyx** [reply]
Hi @iliass1234 
Not really, our tiles are still stored in a server. But we have added a caching strategy
tileCachePath={RNFS.DocumentDirectoryPath + '/Tiles'} in the <UrlTile> component
So the first time app is launched, tiles are loading but after it is quiet smooth.
